### PR TITLE
Hide email on logout

### LINKED ISSUE #1725: User information displayed even after logout in accounts page
When a user is on their "account" page and hits "logout" the website should do a full refresh and flush the email and other sensitive information from the page. Currently if the user is in the "account" page and hits logout the email is still being displayed. 

The screenshot is for CMS but the same issue is observed in politeia

Before logout:
image

After logout:

image

After manual refresh:
image

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
Fixes #1725

### CODE REVIEW

File: src/reducers/models/users.js [hunk 1]
```diff
@@ -61,7 +71,14 @@ const users = (state = DEFAULT_STATE, action) =>
               ["byID", state.currentUserID, "publickey"],
               action.payload.publickey
             )(state),
-          [act.RECEIVE_LOGOUT]: () => onReceiveLogout(state)
+          [act.RECEIVE_LOGOUT]: () => {
+            console.log(state);
```
Comment (amassarwi) [inline src/reducers/models/users.js]: console.log

File: src/reducers/models/users.js [hunk 2]
```diff
@@ -61,7 +71,14 @@ const users = (state = DEFAULT_STATE, action) =>
...
+            return onReceiveLogout(state);
+          },
+          [act.RECEIVE_CMS_LOGOUT]: () => {
+            console.log(state);
```
Comment (amassarwi) [inline src/reducers/models/users.js]: console.log

File: src/actions/api.js [hunk 3]
```diff
@@ -248,22 +248,25 @@ export const onLogin = ({ email, password }) =>
...
   clearPollingPointer();
   clearProposalPaymentPollingPointer();
   dispatch(onSetEmail(""));
   dispatch(act.RESET_USER_SEARCH());
 };
 
-export const onLogout = () =>
+export const onLogout = (isCMS) =>
   withCsrf((dispatch, getState, csrf) => {
+    console.log(isCMS);
```
Comment (amassarwi) [inline src/actions/api.js]: Extra console.log

**Comment by degeri** [comment]
Looks good

**Pull request merged**